src/Common/test-runtime/project.json

```diff
@@ -30,7 +30,7 @@
     "System.Net.WebSockets": "4.0.0-rc3-23809",
     "System.Net.WebSockets.Client": "4.0.0-rc3-23809",
     "System.ObjectModel": "4.0.0",
-    "System.Reflection": "4.0.10",
+    "System.Reflection": "4.0.0",
```

Comment:
Thinking through this I agree.
We are currently working in a Gen 1.3 + world so we should use the stable released version for Gen 1.3 unless required to move to a pre-release version due to development changes.

In this case v4.0.10 is the stable version for Gen 1.3 while v4.0.0 is the stable version for Gens 1.0/1.1/1.2

These were updated correctly in PR #914 so after re-basing this PR these changes are gone.